### ISSUE TITLE
is it possible to use this without using react in your whole app?

### ISSUE THREAD

**Issue by dominictarr** [patchkit/patchkit-image-uploader#1]
question was in title ;)

I think I found the answer to my question here: http://swizec.com/blog/using-react-in-the-real-world/swizec/6710

**Comment by pfrazee** [comment]
Yeah that works, though I do suggest using React if you havent tried it.
